src/hermezjs-typings.d.ts

```diff
@@ -1,5 +1,6 @@
+/* eslint-disable no-empty */
```

Comment:
Why this rule?

Reply:
We want this rule enabled because we want to disallow empty block statements in normal modules. This though, is a type definition module, and here, exceptionally, we need empty blocks, since the syntax to declare a function's signature is as follows:

```
declare function foo(bar: string): string {};
```

If we don't globally disable the rule in this module the linter would complain about each empty block: `{}`.

Reply:
Should we consider disabling this globally as we use this a lot including in catches? @elias-garcia

Reply:
Yeah, I've had to disable it per line in several catches. I'm ok disabling it  globally, not a big deal.

Reply:
Well, in my opinion ESLint rules are to comply with them, not to avoid them in specific lines or files. In this case the `no-empty` rules prevents from having empty blocks of code, like empty `if {}` etc. So in my opinion this rule should stay as it is and we need to delete this comment. If we cannot remove it now because of the refactor we need to do it later.

Reply:
> Should we consider disabling this globally as we use this a lot including in catches?

No longer needed in catches! @elias-garcia showed me a nice trick to troll the linter:

`.catch(() => ({}))`